Podfile

```diff
@@ -34,7 +34,7 @@ def tracks
 end
 
 def wordpress_shared
-  pod 'WordPressShared', '~> 2.0'
+  pod 'WordPressShared', '~> 2.1'
```

Comment:
We are forced to update `WordPressShared` due to the dependency requirement of `WordPressAuthenticator`.